src/main/host/descriptor/socket/mod.rs

```diff
@@ -10,6 +10,7 @@ use crate::utility::event_queue::EventQueue;
 
 use unix::UnixSocketFile;
 
+#[allow(deprecated)]
```

Comment:
It's probably worth having a comment with the link to #2093 in all of these places. Just something like `TODO: https://github.com/shadow/shadow/issues/2093`

Reply:
Added.